src/api/routes/address.ts

```diff
@@ -317,7 +325,15 @@ export function createAddressRouter(db: PgStore, chainId: ChainID): express.Rout
         } else {
           blockHeight = await getBlockHeight(untilBlock, req, res, next, db);
         }
-        const limit = getPagingQueryLimit(ResourceType.Tx, req.query.limit);
+
+        let limit: number;
+        try {
+          limit = getPagingQueryLimit(ResourceType.Tx, req.query.limit);
+        } catch (error: any) {
+          res.status(400).json({ error: error.message });
+          return;
```

Comment:
Do you know why this isn't being handled as expected in the error handling code for `InvalidRequestError` types?

I.e. that `getPagingQueryLimit` throws an error which is supposed to indicate to the rest of the route handling that it's a 400, see:
* https://github.com/hirosystems/stacks-blockchain-api/blob/05d6a9e241d56b05066e8dc4895b4ca0a5086c75/src/api/init.ts#L303
and
* https://github.com/hirosystems/stacks-blockchain-api/blob/05d6a9e241d56b05066e8dc4895b4ca0a5086c75/src/api/init.ts#L325

Reply:
I don't have an immediate answer for that. But probably due to Winston misconfiguration.

Should I take care of that? Or would be better to start working on the new logging using Pino?

Reply:
There's a few other areas depending on the correct behavior for InvalidRequestError handling. I think it makes sense to try to fix that before the pino logging migration, since that might take a while to land, but it can be done in PR after this one (we'd just want to make sure to revert these extra try/catches)